src/main/java/com/google/sps/servlets/MealServlet.java

```diff
@@ -33,21 +44,47 @@
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService(); 
+        // add hard coded meals
+        // 0...4
+        datastore.put(createMealEntity(
+            new Meal(0L, "Fried potato", "Fried potato with mushrooms and onion.",
+                new ArrayList<>(Arrays.asList("potato", "onion", "mushrooms", "oil")), "Main")));
+        datastore.put(createMealEntity(
+            new Meal(1L, "Italian pizza", "Pizza with pineaple, sausage and tomato.",
+                new ArrayList<>(Arrays.asList("flour", "water", "sausage", "tomato", "pineaple", "cheese")), "Pizza")));
+        datastore.put(createMealEntity(
+            new Meal(2L, "Vegetable soup", "Vegetable soup with onion.",
+                new ArrayList<>(Arrays.asList("potato", "onion", "cabbage", "mushrooms", "water", "carrot", "pumpkin")), "Soup")));
+        datastore.put(createMealEntity(
+            new Meal(3L, "Chocolate cake", "Chocolate cake with butter cream and strawberry.",
+                new ArrayList<>(Arrays.asList("flour", "water", "butter", "cocoa powder", "chocolate", "sugar", "strawberry", "eggs")), "Dessert")));
+        datastore.put(createMealEntity(
+            new Meal(4L, "Hot chocolate", "Hot chocolate with sugar and caramel.",
+                new ArrayList<>(Arrays.asList("cocoa powder", "sugar", "milk", "caramel", "vanilla")), "Drinks")));
+        // 5...9
+        datastore.put(createMealEntity(
+            new Meal(5L, "Cheese Omelette", "This tasty omelette is full of cheese.",
+                new ArrayList<>(Arrays.asList("2 eggs", "salt", "butter", "Cheddar cheese", "Mozarella cheese")), "omelette")));
+        datastore.put(createMealEntity(
+            new Meal(6L, "Garlic, Tomato and Cheddar Sandwich", "A delicious sandwich for breakfast and fast food!",
+                new ArrayList<>(Arrays.asList("2 slices brown bread", "clove garlic", "tomato", "Cheddar cheese", "green chili",
+                "olive oil", "salt", "pepper")), "Main")));
+        datastore.put(createMealEntity(
+            new Meal(7L, "Chocolate Chips Pancake", "So delicious and chocolate breakfast.",
+                new ArrayList<>(Arrays.asList("2 eggs", "salt", "butter", "flour", "sugar", "chocolate chips", "milk")), "Pancake")));
+        datastore.put(createMealEntity(
+            new Meal(8L, "Seafood Pizza", "Just from the bottom of the ocean...",
+                new ArrayList<>(Arrays.asList("flour", "salt", "butter", "water", "oil", "shrimp", "tuna", "onion", "ctabstick",
+                "scallops", "fish sticks", "garlic", "tomato sauce")), "Pizza")));
+        datastore.put(createMealEntity(
+            new Meal(9L, "Almond Fudge Brownies", "There is any almond lover's prime recipe for fudge brownies.",
+                new ArrayList<>(Arrays.asList("2 eggs", "sugar", "butter", "milk", "almond", "vanilla", "cocoa powder")), "Dessert")));
+        // TODO: create 5 new ones
```

Comment:
Not sure that this is needed. Probably it's better to start learning how to load everything from wiki.

Reply:
Yeah, I agree. Adding more items manually will probably not enhance the experience too much.

Btw. manually creating data for testing doesn't scale well. A few items are fine, because you can somewhat control their content. However, when one needs to create a larger amount of items, the exact content doesn't matter that much, so it's often easier to just fill the instances with random data.

A classic example of a random text data generator is "lorem ipsum" (there's hundreds of implementations available).
See here for an example of a Java lorem ipsum generator: https://github.com/mdeanda/lorem

Having said that, I think we're about to reach the point where we have to start using some external source of data anyway, so let's not spend too much time adding more hardcoded items.